packages/core/src/requestPool/requestPoolManager.ts

```diff
@@ -110,12 +113,14 @@ class RequestPoolManager {
       interaction: 0,
       thumbnail: 0,
       prefetch: 0,
+      compute: 0,
     };
 
     this.maxNumRequests = {
       interaction: 6,
       thumbnail: 6,
       prefetch: 5,
+      compute: 15,
```

Comment:
Maybe we should change `maxNumRequests`, `requestType`, etc. to a different name because before this change those three first options were related to image requests (download) but now `compute` was added and it is not related to that.